### DISCUSSION TITLE
Mobile browser debugging demo, unable to connect to wallet

### DISCUSSION THREAD

**Discussion by Autumn-Rains-yan** [family/connectkit#186] [Q&A] [answered]
connectkit version: `1.2.2`
I tried nextjs and vite, built the environment locally, and ran it on Android
1: tap Connect Wallet.
2: tap MetaMask,Called up Metamask Dapp.
3: After entering Metamask, sometimes there is no response at all, and sometimes the following prompts appear in Metamask.
![1680586792852](https://user-images.githubusercontent.com/41866020/229697912-d075bf3c-14d6-4678-9156-c4976aef9300.jpg)

[https://docs.family.co/connectkit/try-it-out](url)
The example in the document can be connected to a wallet

**Comment by lochie** [comment, marked as answer]
Looks like a recent update to MetaMask has caused this issue, hopefully they will resolve this on their end soon
https://github.com/MetaMask/metamask-extension/issues/18453
